### PR TITLE
[depends on #240] Importer deletion

### PR DESCRIPTION
**Pull request by jrcastro2**
Allows deletion mode on importer

### CODE REVIEW

**Comment by jrcastro2** [comment]
Note for reviewers: Just review last commit

File: ui/src/overridden/backoffice/Importer/ReportDetails.js [hunk 1]
```diff
@@ -90,6 +90,10 @@ ReportDetailsMiddleColumn.propTypes = {
 };
 
 const ReportDetailsRightColumn = ({ report }) => {
+  let deletedEitemsCount = 0;
+  if (!_isEmpty(report.deleted_eitems)) {
+    report.deleted_eitems.forEach(e => deletedEitemsCount++);
+  }
```
Comment (FlorianCassayre) [inline ui/src/overridden/backoffice/Importer/ReportDetails.js:96]:
```suggestion
  const deletedEitemsCount = !_isEmpty(report.deleted_eitems) ? report.deleted_eitems.length : 0;
```
or
```suggestion
  const deletedEitemsCount = _get(report, 'deleted_eitems.length', 0);
```

File: cds_ils/importer/cli.py [hunk 2]
```diff
@@ -28,16 +29,24 @@ def importer():
...
+    required=True,
+    type=click.Choice(["create", "delete"]),
+    help="Choose the mode",
+)
 @with_appcontext
-def import_from_file(sources, provider, source_type="marcxml"):
+def import_from_file(sources, provider, mode, source_type="marcxml"):
     """Import from file command."""
-    import_from_xml(sources, source_type, provider)
+    import_from_xml(sources, source_type, provider, mode)
```
Comment (FlorianCassayre) [inline cds_ils/importer/cli.py:46]: (minor)
Reorder the arguments for consistency.

**Review by FlorianCassayre — APPROVED** [review]

File: ui/src/config.js [hunk 3]
```diff
@@ -157,5 +157,6 @@ export const config = {
       { key: 'create', text: 'Create', value: 'create' },
       { key: 'delete', text: 'Delete', value: 'delete' },
     ],
+    refreshValue: 5000,
```
Comment (ntarocco) [inline ui/src/config.js:160]: Maybe better naming? `fetch_task_status_interval_secs: 5000` or shorter

**Review by ntarocco — APPROVED** [review]

**Review by kprzerwa — APPROVED** [review]
all comments to be fixed in another PR

**Pull request merged**